src/css/parse.js

```diff
@@ -1203,75 +1249,19 @@ function checkFunction(i) {
  * @return {Node}
  */
 function getFunction() {
-  let type = NodeType.FunctionType;
-  let token = tokens[pos];
-  let line = token.ln;
-  let column = token.col;
-  let ident = getIdent();
-  let content = [ident];
+  const type = NodeType.FunctionType;
+  const token = tokens[pos];
+  const line = token.ln;
+  const column = token.col;
+  let content = [getIdent()];
+  let body = getArguments();
 
-  content.push(getArguments());
+  content.push(body);
```

Comment:
That made me laugh a bit.
You've replaced `let ident = getIdent(); let content = [ident]` with `let content = [getIdent()]`. Okay, I suppose it doesn't make sense to introduce a variable just for one line, right? But then on the next line you've replaced `content.push(getArguments())` with `let body = getArguments(); content.push(body)` which is _exactly the opposite of your first change_ :)

How about `const content = [getIdent(), getArguments()]`?